data/sql/updates/pending_db_world/bad-ai.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE `creature_template` SET `AIName` = '' WHERE `entry` IN (2307, 11940, 11941, 11943, 19064, 32743, 37072);
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (2307, 11940, 11941, 11943, 19064, 32743, 37072);
```

Comment:
You need to delete with SourceGroup

Reply:
Did you mean `source_type`? If so, done.

Reply:
Sure, SourceGroup must be conditions. Cant remember everything while on the road x)